pkg/kubectl/cmd/expose.go

```diff
@@ -175,6 +176,9 @@ func RunExpose(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []str
 		}
 		params["labels"] = kubectl.MakeLabels(labels)
 	}
+	if cmdutil.GetFlagBool(cmd, "headless") {
```

Comment:
quick question,

I ran `hack/update-all.sh`, and there were errors (from osx) - Not sure if these are known issues:
```
hack/update-all.sh 
Running in the silent mode, run with -v if you want to see script logs.
Updating codecgen
readlink: illegal option -- f
usage: readlink [-n] [file ...]
...
hack/../hack/update-codecgen.sh: line 85: godep: command not found
!!! Error in hack/../hack/update-codecgen.sh:85
  'godep go build -o "${CODECGEN}" github.com/ugorji/go/codec/codecgen' exited with status 127
Call stack:
  1: hack/../hack/update-codecgen.sh:85 main(...)
Exiting with status 1
FAILED
Updating generated-conversions
Updating generated-deep-copies
Updating generated-docs
Updating generated-swagger-docs
hack/../hack/update-generated-swagger-docs.sh: line 49: godep: command not found
go: GOPATH entry is relative; must be absolute path: "".
Run 'go help gopath' for usage.
!!! Error in hack/../hack/update-generated-swagger-docs.sh:26
  'GOPATH=$(godep path):$GOPATH go run cmd/genswaggertypedocs/swagger_type_docs.go -s "pkg/$(kube::util::group-version-to-pkg-path "${group_version}")/types.go" -f - >> "$TMPFILE"' exited with status 2
Call stack:
  1: hack/../hack/update-generated-swagger-docs.sh:26 generate_version(...)
  2: hack/../hack/update-generated-swagger-docs.sh:65 main(...)
Exiting with status 1
FAILED
Updating swagger-spec

etcd must be in your PATH

!!! Error in hack/../hack/update-swagger-spec.sh:31
  '"${KUBE_ROOT}/hack/after-build/update-swagger-spec.sh" "$@"' exited with status 1
Call stack:
  1: hack/../hack/update-swagger-spec.sh:31 main(...)
Exiting with status 1
```

When I try to commit, it also deleted a couple of files - should these deletions be committed as well?
```
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   contrib/completions/bash/kubectl
	modified:   docs/man/man1/kubectl-expose.1
	modified:   docs/user-guide/kubectl/kubectl_expose.md
	deleted:    pkg/api/unversioned/types_swagger_doc_generated.go
	deleted:    pkg/api/v1/types_swagger_doc_generated.go
	deleted:    pkg/apis/extensions/v1beta1/types_swagger_doc_generated.go
```